config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
> However, I am concerned about whether or not this is the right approach.
> Do we know what is in the session causing us to exceed the current 1MB max? It may be easy to find and remove.

I agree.  It feels like a band-aid that might prevent us from fixing the problem.

> I was hoping that memcached could be the first part of the appliance that could be extracted to an independent container. This level of configuration makes that more difficult.

Yes, I agree, this would make it harder.  Presumably, if we add more configuration of memcached into manageiq, it would be expected we support it even after we start splitting up the monolithic container.  

So, looking at [dalli's readme](https://github.com/petergoldstein/dalli/tree/695e0c0a0811c5bf219fd01bb9aaf933012ed2f7#configuration), it looks like dalli should still allow storing more than 1 MB but with a warning:

```
value_max_bytes: The maximum size of a value in memcached. Defaults to 1MB, this can be increased with memcached's -I parameter. You must also configure Dalli to allow the larger size here.

error_when_over_max_size: Boolean. If true, Dalli will throw a Dalli::ValueOverMaxSize exception when trying to store data larger than value_max_bytes. Defaults to false, meaning only a warning is logged.
```

This default behavior is fine with me, it should log a warning but still allow it.  Therefore @nenioscio, you should only need to change the `memcache_server_opts` to:

```
memcache_server_opts: "-l 127.0.0.1 -I 3M"
```

@nenioscio Can you confirm this?  We shouldn't add configuration if it's not required.  Thanks.